packages/just-scripts/src/tasks/jestTask.ts

```diff
@@ -31,12 +31,25 @@ export interface JestTaskOptions {
 
 export function jestTask(options: JestTaskOptions = {}): TaskFunction {
   const jestConfigFile = resolveCwd('./jest.config.js');
+  const packageConfigFile = resolveCwd('./package.json');
 
   return function jest() {
     const jestCmd = resolve('jest/bin/jest.js');
     const configFile = options.config || jestConfigFile;
+    const configFileExists = configFile && existsSync(configFile);
```

Comment:
You actually don't need to do the exists check separately either, just do `readPackageJson(process.cwd())` and it will return undefined if the file doesn't exist.

Reply:
Do you mean I don't need to check for existence of the package file? The code you referenced checks for the jest config file.

As for calling process.cwd(), I noticed that the resolveCwd() calls are made when jestTask() is called, not when the function runs. I'm thinking I need to at least get the cwd() at that time, in case the directory context is different when the task runs.